### ISSUE TITLE
D3FEND 1.0 Content Deprecation Strategy

### ISSUE THREAD

**Issue by netfl0** [d3fend/d3fend-ontology#313]
https://github.com/d3fend/d3fend-ontology/compare/develop...313-d3fend-10-content-deprecation-strategy

Draft strategy.

**Comment by hack-sentinel** [comment]
Everything seems fine except the prepend of DEPRECATED to the rdfs:label, which people might be using in the grace period.  For some folks this change would make their existing stuff if they have apps or anything displaying label unusable not just not recommended for use/future use...

I would change:
   Prepending the string `DEPRECATED ` to `rdfs:label` and any existing `rdfs:comment` .
to just:
   Prepending the string `DEPRECATED ` to any existing `rdfs:comment` .

**Comment by netfl0** [comment]
I agree it would cause heartburn for developers auto-ingesting and then rendering. However, the flip side of the argument is that: 
 - it might be the only way they see and know a deprecation is coming. 
 - I disagree it would be unusable, depending on the app, if they were to have a limited character display length, yes it is possible it would push the entity name too far to the right. However, we also have some really long entity names so then those would also be unusable in such an application.
 - This is default protege behavior which gives all users a strong indication the entity is going to be deprecated.
 
 I still recommend we keep this approach.

**Comment by aamedina** [comment]
Seems reasonable to me. Does this link basically go over everything? (They just call deprecate obsolete) https://oboacademy.github.io/obook/howto/obsolete-term/
